tests/test_nko.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+from xnum import convert, NumeralSystem
+
+TEST_CASE_NAME = "Nko tests"
+NKO_DIGITS = "߀߁߂߃߄߅߆߇߈߉"
+
+
+CONVERSION_CASES = {
+    NumeralSystem.ARABIC_INDIC: "٠١٢٣٤٥٦٧٨٩",
+    NumeralSystem.ENGLISH: "0123456789",
+    NumeralSystem.ENGLISH_FULLWIDTH: "０１２３４５６７８９",
+    NumeralSystem.ENGLISH_SUBSCRIPT: "₀₁₂₃₄₅₆₇₈₉",
+    NumeralSystem.ENGLISH_SUPERSCRIPT: "⁰¹²³⁴⁵⁶⁷⁸⁹",
+    NumeralSystem.ENGLISH_DOUBLE_STRUCK: "𝟘𝟙𝟚𝟛𝟜𝟝𝟞𝟟𝟠𝟡",
+    NumeralSystem.ENGLISH_BOLD: "𝟎𝟏𝟐𝟑𝟒𝟓𝟔𝟕𝟖𝟗",
+    NumeralSystem.ENGLISH_MONOSPACE: "𝟶𝟷𝟸𝟹𝟺𝟻𝟼𝟽𝟾𝟿",
+    NumeralSystem.ENGLISH_SANS_SERIF: "𝟢𝟣𝟤𝟥𝟦𝟧𝟨𝟩𝟪𝟫",
+    NumeralSystem.ENGLISH_SANS_SERIF_BOLD: "𝟬𝟭𝟮𝟯𝟰𝟱𝟲𝟳𝟴𝟵",
+    NumeralSystem.PERSIAN: "۰۱۲۳۴۵۶۷۸۹",
+    NumeralSystem.HINDI: "०१२३४५६७८९",
+    NumeralSystem.BENGALI: "০১২৩৪৫৬৭৮৯",
+    NumeralSystem.THAI: "๐๑๒๓๔๕๖๗๘๙",
+    NumeralSystem.KHMER: "០១២៣៤៥៦៧៨៩",
+    NumeralSystem.BURMESE: "၀၁၂၃၄၅၆၇၈၉",
+    NumeralSystem.TIBETAN: "༠༡༢༣༤༥༦༧༨༩",
+    NumeralSystem.GUJARATI: "૦૧૨૩૪૫૬૭૮૯",
+    NumeralSystem.ODIA: "୦୧୨୩୪୫୬୭୮୯",
+    NumeralSystem.TELUGU: "౦౧౨౩౪౫౬౭౮౯",
+    NumeralSystem.KANNADA: "೦೧೨೩೪೫೬೭೮೯",
+    NumeralSystem.GURMUKHI: "੦੧੨੩੪੫੬੭੮੯",
+    NumeralSystem.LAO: "໐໑໒໓໔໕໖໗໘໙",
+    NumeralSystem.NKO: NKO_DIGITS
```

Comment:
Since this font is RTL, it's good to have a different unit test that capture that special charactersitic as well.

Reply:
Since handling RTL/LTR directionality depends on the environment (e.g., most console and terminal environments don't support RTL), there's nothing more we can test here.